composer.json

```diff
@@ -39,7 +39,10 @@
 		"symfony/uid": "^6.4",
 		"symfony/yaml": "^6.4",
 		"twig/twig": "^3.14",
-		"web-token/jwt-framework": "^3"
+		"web-token/jwt-framework": "^3",
+      	"ext-simplexml": "*",
+		"ext-libxml": "*",
+		"ext-dom": "*"
```

Comment:
Did you check these were standard enabled in Nextcloud (they are, but please check before requiring extra extensions)

Reply:
![image](https://github.com/user-attachments/assets/aa314661-70ec-4908-9c21-edbadbf2a7a5)
![image](https://github.com/user-attachments/assets/24fb7561-3a20-4caf-967b-5e76f8afe399)